README.md

```diff
@@ -43,19 +43,25 @@ The repository is laid out as follows:
 
 * `bin` - Scripts used by Cromwell configuration settings. Add this to `$PATH` prior to using configurations  in `conf` with Cromwell.
 * `conf` - Cromwell configuration files created for various environments that we use across our team. Feel free to use/fork/suggest improvements.
-* `docker` - Dockerfiles used in our workflows. All docker images are published to [Docker Hub](https://hub.docker.com/u/stjudecloud).
-  * At the time of writing, there is only one Docker image heavily in use, which is our [bioinformatics base image](./docker/bioinformatics-base/Dockerfile).
+* `docker` - Dockerfiles used in our workflows. All docker images are published to [Docker Hub](https://hub.docker.com/u/stjudecloud) as a part of our CI and are versioned.
```

Comment:
The department guidance is moving to use GitHub packages rather than Docker hub. We may want to consider doing the same.

Reply:
Is there a reason for preferring GitHub packages? Because we initially used GitHub packages till we learned releases couldn't be deleted or overwritten. The way our CI works, we create branch specific images which would be permanent in GH, but removable in Docker hub.

Reply:
One of the primary motivating factors is the pull rate limitation. In Docker hub, the limit is applied to the user, not the repository. In GitHub packages, the transfers are limited at the repository level and are unlimited for public repos.

Reply:
That makes a lot of sense. I'll add switching to GH packages on Trello